### DISCUSSION TITLE
拖拽放大，然后点击按钮缩小，不能缩到1，只能缩到1.1

### DISCUSSION THREAD

**Discussion by du937935598** [nanxiaobei/antd-img-crop#289] [Q&A] [answered]
![image](https://github.com/nanxiaobei/antd-img-crop/assets/17040713/d7c154e7-aa73-4314-bac4-697066b8b577)

这里代码逻辑错了，需要小于等于

**Comment by du937935598** [comment, marked as answer]
disabled remove ZOOM_STEP

**Comment by nanxiaobei** [comment]
感谢反馈，`4.22.0` 已修复。
